src/components/accession2/view/DetailPanel.tsx

```diff
@@ -158,8 +158,10 @@ export default function DetailPanel(props: DetailPanelProps): JSX.Element {
 
               {accession.collectionSiteNotes && (
                 <Box marginTop={2} display='flex'>
-                  <Icon name='iconFile' className={classes.folderIcon} />
-                  <Typography>{accession.collectionSiteNotes}</Typography>
+                  <Typography>
```

Comment:
center align?

Reply:
The typography is not display flex, so I can't set up align center. I could set up a line height, but I like more how it looks without line height

Reply:
Ok sounds good.